features/180atp/180-days_ATP_feature_instructions.md

```diff
@@ -0,0 +1,91 @@
+# How to use 180-days ATP
+
+## Introduction
+
+180-days ATP is a more powerful version of the previous 7-days ATP feature.  
+It allows you to schedule on-hand changes and calculate available-to-promise (ATP) for up to 180 days in advance.
+
+## Turning on the feature
+
+By default, the ATP feature is limited to 7 days.  
+To turn on the 180-days ATP feature, follow these steps:
+
+1. Turn on the ATP feature.
+2. Turn on the 180-days ATP feature.
+3. Update Configuration.
+
+<img src="./ATP%20Feature%20Switch.png" alt="ATP feature settings" style="width:30%">  
+
+<div style="background-color: #e6f3ff; padding: 10px; border-radius: 10px;">
+<strong>Tips:</strong><br>
+1. 180-days ATP is only supported when entity version is 2.<br>
+If you are using Legacy UI, we will soon provide a help doc about how to upgrade your entity version and migrate your ATP configuration.<br>
+<br>
+2. Please note that the 7-day ATP and 180-days ATP features are separate and independent from each other. Any schedule changes created or modified using the 7-day ATP feature will not take effect when the 180-day ATP feature is turned on.
+</div>
+
+## Configuring the feature
+
+There are three settings you should configure for the ATP feature:
+
+1. Max Schedule Period (Days):  
+   This is how long you can schedule changes for. The maximum value for this field is 180 days.  
+   By default, it is set to 30 days, which means you can schedule changes for up to 30 days from today.
+
+2. Schedule Measures: <br>
+   Schedule Measures could be an existing calculated measure or you can create a new calculated measure.  
+   The ATP value will be provided for defined calculated measures, based on the scheduled changes of component physical measures.
+
+   <img src="./Schedule%20Measures.png" alt="Schedule Measures Configuration" style="width:85%">  
+
+3. ATP Index Set Configuration:  
+   This setting is similar to the "Product Index Hierarchy" which allows you to group your query results by specific dimensions.  
```

Comment:
Not quite, for 'product index hierarchy' you are allowed to query not obeying the hierarchy. Whereas for this feature seems you're coerced to do so...